tools/realsense-viewer/CMakeLists.txt

```diff
@@ -167,12 +172,13 @@ endif()
 
 # config-ui
 if(WIN32)
-    add_executable(realsense-viewer WIN32
-        ${RS_VIEWER_CPP}
-        ${ELPP_SOURCES}
-        ${CMAKE_CURRENT_SOURCE_DIR}/res/resource.h
-        ${CMAKE_CURRENT_SOURCE_DIR}/res/realsense-viewer.rc
-        ../../common/windows-app-bootstrap.cpp
+    target_sources( ${PROJECT_NAME}
+        PRIVATE
+            ${RS_VIEWER_CPP}
+            ${ELPP_SOURCES}
```

Comment:
We don't need it now right?

Reply:
Same comment relevant to the DQT CMakelists file

Reply:
Correct... removed